### PR TITLE
fix opmapper test not support inplace var bug

### PR DESCRIPTION
**Pull request by thisjiang**
修复OpMapper跑不了`BatchNorm`算子，因为`BatchNorm`中含有inplace变量。同时在OpMapper test中添加了`batch_norm`算子的单测。

### CODE REVIEW

**Review by SunNy820828449 — APPROVED** [review]
LGTM
